.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,55 @@
+name: Error Code Reference Publisher
```

Comment:
@sayantan1413 I've made a few changes to the one currently in meshkit. Please take a look and update accordingly. Thanks : ). https://github.com/DelusionalOptimist/meshkit/blob/error-codes-final/.github/workflows/error-codes-updater.yaml